.github/workflows/build.yml

```
@@ -0,0 +1,31 @@
+name: build
+on:
+  push:
+  pull_request:
+    branches: [master]
+  workflow_dispatch:
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      max-parallel: 5
+      matrix:  # TODO: Add all the following OSes...
+        # os: [macos-13, macos-latest, ubuntu-latest, ubuntu-24.04-arm, windows-latest, windows-2025]
+        os: [macos-13, ubuntu-latest, ubuntu-24.04-arm]  # macOS on Intel and Ubuntu builds succeed.
```

GitHub Actions will drop `macos-13` in 3 days.
```suggestion
        # os: [macos-15-intel, macos-latest, ubuntu-latest, ubuntu-24.04-arm, windows-latest, windows-2025]
        os: [macos-15-intel, ubuntu-latest, ubuntu-24.04-arm]  # macOS on Intel and Ubuntu builds succeed.
```